microshift_release_notes/microshift-4-13-release-notes.adoc

```diff
@@ -59,8 +59,10 @@ This release adds improvements related to the following components and concepts.
 //[id="microshift-4-13-networking"]
 //=== Networking
 
-//[id="microshift-4-13-storage"]
-//=== Storage
+[id="microshift-4-13-storage"]
+=== Storage
+
+* {product-title} configures LVMS more dynamically based on the Volume Groups present. For more details, read the xref:../microshift_storage/microshift-storage-plugin-overview.adoc#microshift-lvms-system-requirements_{context}[LVMS system requirements] section of this documentation.
```

Comment:
```suggestion
* {product-title} configures LVMS more dynamically based on the Volume Groups present. For more details, read the xref:../microshift_storage/microshift-storage-plugin-overview.adoc#microshift-lvms-system-requirements[LVMS system requirements] section of this documentation.
```

Reply:
[id="lvms-system-requirements_{context}"] That's the id of the module I can see in the 4.13 branch. The file name of the module was changed, but the id didn't get updated.

Reply:
I think I worked out the right expanded form with the context value. Let's see how it works.